### PR TITLE
Cleanup native voting system code #1071 Use lifeLongDelegatesProtocol

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1071
5、Use lifeLongDelegatesProtocol by default to feed delegate info to make blockchain be able to run by default.

### CODE REVIEW

**Comment by zjshen14** [comment]
Please fix make run per offline discussion

**Comment by zjshen14** [comment]
@lzxm160, When `registerDefaultProtocols`, I think if the consensus mode is not roll-dpos, we shouldn't register poll, rewarding protocols at all. These all don't make sense to standalone consensus.

**Comment by lzxm160** [comment]
> @lzxm160, When `registerDefaultProtocols`, I think if the consensus mode is not roll-dpos, we shouldn't register poll, rewarding protocols at all. These all don't make sense to standalone consensus.

poll protocol removed,rewarding protocal keeped because it caused more errors

File: blockchain/blockchain.go [hunk 1]
```diff
@@ -1088,34 +1088,37 @@ func (bc *blockchain) pickAndRunActions(ctx context.Context, actionMap map[strin
...
-	// generate delegates for next round
-	skip, putPollResult, err := bc.createPutPollResultAction(raCtx.BlockHeight)
-	switch errors.Cause(err) {
-	case nil:
-		if !skip {
-			receipt, err := ws.RunAction(raCtx, putPollResult)
-			if err != nil {
-				return hash.ZeroHash256, nil, nil, err
+	var lastBlkHeight uint64
+	if bc.config.Consensus.Scheme != config.StandaloneScheme {
```
[the remark this answers is not in the document]
Comment (lzxm160) [reply blockchain/blockchain.go]: there's another scheme need register poll protocol

**Pull request merged**